web/src/utils.js

```diff
@@ -25,7 +25,7 @@
  *
  * @param {Array} collection - the collection to be filtered
  * @param {function} filter - the function to be used as filter
- * @return {[Array, Array]} the new array with grouped elements
+ * @return {Array[]} a pair of arrays, [passing, failing]
```

Comment:
Nice!

We _fixed_ it by using `<Array.<Array, Array>` but the change has not been uploaded yet. That said, I like more your approach. Thank you!